lib/ansible/modules/system/iptables.py

```diff
@@ -399,8 +399,14 @@ def construct_rule(params):
         False)
     append_match(rule, params['comment'], 'comment')
     append_param(rule, params['comment'], '--comment', False)
-    append_match(rule, params['ctstate'], 'state')
-    append_csv(rule, params['ctstate'], '--state')
+    if params['ctstate']:
```

Comment:
see here, http://unix.stackexchange.com/questions/108169/what-is-the-difference-between-m-conntrack-ctstate-and-m-state-state. The --state parameter is being replaced by conntrack.